app/views/admin/movies/_form.html.erb

```diff
@@ -8,5 +8,13 @@
   <%= f.text_field :time, class: "form-control" %>
   <%= f.label :picture %>
   <%= f.file_field :picture, class: "form-control" %>
+  <div class="gender">
+    <%= f.radio_button :status, :is_showing %>
+    <%= f.label :Showing %>
+    <%= f.radio_button :status, :is_comming %>
+    <%= f.label :Comming %>
+    <%= f.radio_button :status, :ended %>
+    <%= f.label :Ended %>
```

Comment:
tìm hiểu sử dụng enum kết hợp gen radio button hì, chứ như ri chị thêm giá trị enum, hoặc xóa đi bớt thì sửa hơi bị nhiuf

https://stackoverflow.com/questions/25068493/how-do-you-use-active-record-enum-radio-buttons-in-a-form
https://www.rubydoc.info/github/plataformatec/simple_form/SimpleForm/FormBuilder:collection_check_boxes